CorsixTH/Lua/entity_map.lua

```diff
@@ -45,8 +45,8 @@ end
 @param y (integer) the y coordinate of the entity
 @param entity (Entity) the entity to be added to the map in the (x,y) position]]
 function EntityMap:addEntity(x,y,entity)
-  local function add_entity_to_table(entity,entity_table)
-    entity_table[#entity_table+1] = entity
+  local function add_entity_to_table(enty, entity_table)
```

Comment:
Promote the local function to the top level. It doesn't use the closure and that avoids renaming the variable.